src/com/xilinx/rapidwright/rwroute/RouteNodeGraph.java

```diff
@@ -536,4 +573,21 @@ public boolean isAccessible(RouteNode child, Connection connection) {
         return Math.abs(childTile.getTileYCoordinate() - sinkTile.getTileYCoordinate()) <= 1;
     }
 
+    protected boolean allowRoutethru(Node head, Node tail) {
+        if (!Utils.isCLB(tail.getTile().getTileTypeEnum())) {
+            return false;
+        }
+
+        if (!RouteThruHelper.isRouteThruPIPAvailable(design, head, tail)) {
+            return false;
+        }
+        assert(PIP.getArbitraryPIP(head, tail).isRouteThru());
+
+        BitSet bs = muxWires.get(tail.getTile().getTileTypeEnum());
+        if (bs != null && bs.get(tail.getWire())) {
+            // Disallow * -> [A-H]MUX routethrus
```

Comment:
Can you provide a bit more documentation on why we should disallow `[A-H]MUX` routethrus?

Reply:
Expanded the comment.